source/jsonnet/england-wales/ccs/blocks/individual/ethnic_group.jsonnet

```diff
@@ -9,11 +9,23 @@ local proxyTitle = {
   ],
 };
 
-local englandInstruction = 'Tell the respondent to turn to <strong>Showcard 9E</strong> or show them the answer options below';
-local walesInstruction = 'Tell the respondent to turn to <strong>Showcard 9W</strong> or show them the answer options below';
+local englandInstruction = 'Tell the respondent to turn to <strong>Showcard 9E</strong> or show them the options below';
+local walesInstruction = 'Tell the respondent to turn to <strong>Showcard 9W</strong> or show them the options below';
 
+local englandDescription = 'Includes British, Northern Irish, Irish, Gypsy, Irish Traveller, Roma or any other White background';
+local walesDescription = 'Includes Welsh, British, Northern Irish, Irish, Gypsy, Irish Traveller, Roma or any other White background';
+
+local englandAsianOption = 'Asian or Asian British';
+local walesAsianOption = 'Asian, Asian Welsh or Asian British';
+
+local englandBlackOption = 'Black, Black British, Caribbean or African';
+local walesBlackOption = 'Black, Black Welsh, Black British, Caribbean or African';
+
+local question(title, instruction, region_code) = (
+  local regionDescription = if region_code == 'GB-WLS' then walesDescription else englandDescription;
```

Comment:
I am not sure we should have 6 'if' statements checking for the language, it should only do it once. I reckon we should put all the strings in 2 dicts (Eng and Welsh), then pass the correct dict into the question. It would also clean up all the local variables. I won't share it on here, but will send my code

Reply:
I changed some naming e.g. _wales/englandStrings_, _regionStrings_, _instruction_ and added same logic to _ethnic_group_asian_ and _ethnic_group_black_. It cleans up a lot. Thanks👍